app/views/layouts/_support_links.html.erb

```diff
@@ -0,0 +1,5 @@
+<h2 class="govuk-heading-m">Support and guidance</h2>
+<p class="govuk-body-s">
+  If you have a question, or you've had a problem using this service, please contact us at
+  <%= render MailToSupportComponent.new("continuing-professional-development@digital.education.gov.uk") %>
```

Comment:
Without the email in text, does it render the same? 
So `<%= render MailToSupportComponent.new() %>`?